Cannot build on `aarch64-pc-windows-msvc`

```
error: failed to run custom build command for `ring v0.16.20`

Caused by:
  process didn't exit successfully: `C:\Users\igor\Documents\Workspaces\playground\schema-registry-cli\target\debug\build\ring-8b996bf917a3065b\build-script-build` (exit code: 101)
  --- stdout
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-pc-windows-msvc")
  HOST = Some("aarch64-pc-windows-msvc")
  cargo:rerun-if-env-changed=CC_aarch64-pc-windows-msvc
  CC_aarch64-pc-windows-msvc = None
  cargo:rerun-if-env-changed=CC_aarch64_pc_windows_msvc
  CC_aarch64_pc_windows_msvc = None
  cargo:rerun-if-env-changed=HOST_CC
  HOST_CC = None
  cargo:rerun-if-env-changed=CC
  CC = None
  cargo:rerun-if-env-changed=CFLAGS_aarch64-pc-windows-msvc
  CFLAGS_aarch64-pc-windows-msvc = None
  cargo:rerun-if-env-changed=CFLAGS_aarch64_pc_windows_msvc
  CFLAGS_aarch64_pc_windows_msvc = None
  cargo:rerun-if-env-changed=HOST_CFLAGS
  HOST_CFLAGS = None
  cargo:rerun-if-env-changed=CFLAGS
  CFLAGS = None
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("neon")
  DEBUG = Some("true")
  montgomery.c
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\internal.h(191): error C2065: 'BN_ULLONG': undeclared identifier
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\internal.h(191): error C2146: syntax error: missing ';' before identifier 'result'
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\internal.h(191): warning C4555: result of expression not used
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\internal.h(191): error C2065: 'result': undeclared identifier
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\internal.h(191): error C2146: syntax error: missing ';' before identifier 'a'
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\internal.h(191): warning C4552: '*': result of expression not used
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\internal.h(192): error C2065: 'result': undeclared identifier
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\internal.h(193): error C2065: 'result': undeclared identifier
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\internal.h(193): warning C4293: '>>': shift count negative or too big, undefined behavior
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\../../limbs/limbs.inl(38): warning C4163: '_addcarry_u64': not available as an intrinsic function
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\../../limbs/limbs.inl(38): warning C4163: '_subborrow_u64': not available as an intrinsic function
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\../../limbs/limbs.inl(62): warning C4013: '_addcarry_u64' undefined; assuming extern returning int
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\../../limbs/limbs.inl(62): warning C4242: '=': conversion from 'int' to 'Carry', possible loss of data
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\../../limbs/limbs.inl(76): warning C4242: '=': conversion from 'int' to 'Carry', possible loss of data
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\../../limbs/limbs.inl(92): warning C4013: '_subborrow_u64' undefined; assuming extern returning int
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\../../limbs/limbs.inl(92): warning C4242: '=': conversion from 'int' to 'Carry', possible loss of data
  C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\crypto\fipsmodule\bn\../../limbs/limbs.inl(106): warning C4242: '=': conversion from 'int' to 'Carry', possible loss of data

  --- stderr
  running "C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.35.32215\\bin\\HostX86\\arm64\\cl.exe" "-nologo" "-MD" "-Z7" "-Brepro" "-I" "include" "-W4" "/GS" "/Gy" "/EHsc" "/GR-" "/Zc:wchar_t" "/Zc:forScope" "/Zc:inline" "/Zc:rvalueCast" "/sdl" "/Wall" "/wd4127" "/wd4464" "/wd4514" "/wd4710" "/wd4711" "/wd4820" "/wd5045" "/Od" "/RTCsu" "-DNDEBUG" "-c" "/FoC:\\Users\\igor\\Documents\\Workspaces\\playground\\schema-registry-cli\\target\\debug\\build\\ring-67a383fea41c2a3d\\out\\montgomery.obj" "crypto/fipsmodule/bn/montgomery.c"
  thread 'main' panicked at 'execution failed', C:\Users\igor\.cargo\registry\src\github.com-1ecc6299db9ec823\ring-0.16.20\build.rs:656:9
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

--- a/schema-registry-api/CHANGELOG.md
+++ b/schema-registry-api/CHANGELOG.md
@@ -5,6 +5,31 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Bug Fixes (BREAKING)
+
+ - <csr-id-3972b096e3d922cec5a6dad154b52b811a613e0c/> 🐛 Let user choose the TLS reqwest feature
+
+### Commit Statistics
+
+<csr-read-only-do-not-edit/>
+
+ - 1 commit contributed to the release.
+ - 7 days passed between releases.
+ - 1 commit was understood as [conventional](https://www.conventionalcommits.org).
+ - 0 issues like '(#ID)' were seen in commit messages
+
+### Commit Details
+
+<csr-read-only-do-not-edit/>
+
+<details><summary>view details</summary>
+
+ * **Uncategorized**
+    - 🐛 Let user choose the TLS reqwest feature ([`3972b09`](https://github.com/ilaborie/schema-registry-cli/commit/3972b096e3d922cec5a6dad154b52b811a613e0c))
+</details>
+
 ## 1.0.0 (2023-04-23)
 
 <csr-id-4153bba84c647a891383c17e997873b38d722551/>
@@ -18,13 +43,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
  - <csr-id-4153bba84c647a891383c17e997873b38d722551/> ♻️ Borrow instead of move
 
+### Chore
+
+ - <csr-id-e163240ccfce3c65780bf9c606335db006c26219/> 🏷️ Release schema-registry-api v1.0.0
+
 ### Commit Statistics
 
 <csr-read-only-do-not-edit/>
 
- - 2 commits contributed to the release.
+ - 3 commits contributed to the release.
  - 7 days passed between releases.
- - 2 commits were understood as [conventional](https://www.conventionalcommits.org).
+ - 3 commits were understood as [conventional](https://www.conventionalcommits.org).
  - 0 issues like '(#ID)' were seen in commit messages
 
 ### Commit Details
@@ -34,6 +63,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 <details><summary>view details</summary>
 
  * **Uncategorized**
+    - 🏷️ Release schema-registry-api v1.0.0 ([`e163240`](https://github.com/ilaborie/schema-registry-cli/commit/e163240ccfce3c65780bf9c606335db006c26219))
     - ✨ CLI for schema registry ([`61b0267`](https://github.com/ilaborie/schema-registry-cli/commit/61b026741b67ff31692965d218c89d11eb2658d3))
     - ♻️ Borrow instead of move ([`4153bba`](https://github.com/ilaborie/schema-registry-cli/commit/4153bba84c647a891383c17e997873b38d722551))
 </details>

--- a/schema-registry-api/CHANGELOG.md
+++ b/schema-registry-api/CHANGELOG.md
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## 2.0.0 (2023-05-01)
+
+### Documentation
+
+ - <csr-id-733a965c564a53eb4dd899e46af6dbcb24dd102a/> 📄 Update changelogs
 
 ### Bug Fixes (BREAKING)
 
@@ -15,9 +19,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <csr-read-only-do-not-edit/>
 
- - 1 commit contributed to the release.
+ - 2 commits contributed to the release.
  - 7 days passed between releases.
- - 1 commit was understood as [conventional](https://www.conventionalcommits.org).
+ - 2 commits were understood as [conventional](https://www.conventionalcommits.org).
  - 0 issues like '(#ID)' were seen in commit messages
 
 ### Commit Details
@@ -27,12 +31,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 <details><summary>view details</summary>
 
  * **Uncategorized**
+    - 📄 Update changelogs ([`733a965`](https://github.com/ilaborie/schema-registry-cli/commit/733a965c564a53eb4dd899e46af6dbcb24dd102a))
     - 🐛 Let user choose the TLS reqwest feature ([`3972b09`](https://github.com/ilaborie/schema-registry-cli/commit/3972b096e3d922cec5a6dad154b52b811a613e0c))
 </details>
 
 ## 1.0.0 (2023-04-23)
 
 <csr-id-4153bba84c647a891383c17e997873b38d722551/>
+<csr-id-e163240ccfce3c65780bf9c606335db006c26219/>
 
 ### New Features
 

--- a/schema-registry-api/Cargo.toml
+++ b/schema-registry-api/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "schema-registry-api"
-version = "1.0.0"
+version = "2.0.0"
 authors = ["Igor Laborie <ilaborie@gmail.com>"]
 description = "REST API for a schema-registry"
 license = "MIT OR Apache-2.0"

--- a/schema-registry-api/Cargo.toml
+++ b/schema-registry-api/Cargo.toml
@@ -16,12 +16,19 @@ homepage = "https://github.com/ilaborie/schema-registry-cli/schema-registry-api"
 repository = "https://github.com/ilaborie/schema-registry-api.git"
 
 [features]
+default = []
+
+# Feature from reqwest
+native-tls = ["reqwest/native-tls"]
+native-tls-vendored = ["reqwest/native-tls-vendored"]
+native-tls-alpn = ["reqwest/native-tls-alpn"]
+rustls-tls = ["reqwest/rustls-tls"]
+rustls-tls-manual-roots = ["reqwest/rustls-tls-manual-roots"]
+rustls-tls-webpki-roots = ["reqwest/rustls-tls-webpki-roots"]
+rustls-tls-native-roots = ["reqwest/rustls-tls-native-roots"]
 
 [dependencies]
-reqwest = { version = "0.11.16", features = [
-    "json",
-    "rustls-tls-native-roots",
-], default-features = false }
+reqwest = { version = "0.11.16", features = ["json"], default-features = false }
 reqwest-middleware = "0.2.1"
 reqwest-tracing = "0.4.1"
 serde = { version = "1", features = ["derive"] }

--- a/schema-registry-cli/CHANGELOG.md
+++ b/schema-registry-cli/CHANGELOG.md
@@ -5,22 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0.0 (2023-04-23)
+## Unreleased
 
 ### Chore
 
- - <csr-id-25ebd97c1c43e01754b11c44ab08a0bf37286a8c/> 🍽️ prepare cli release
-
-### New Features
-
- - <csr-id-61b026741b67ff31692965d218c89d11eb2658d3/> ✨ CLI for schema registry
+ - <csr-id-e1bf358ecc8f34464c657655322434793cb698b8/> 🏷️ Release schema-registry-api v2.0.0, safety bump schema-registry-cli v2.0.0
 
 ### Commit Statistics
 
 <csr-read-only-do-not-edit/>
 
- - 2 commits contributed to the release.
- - 2 commits were understood as [conventional](https://www.conventionalcommits.org).
+ - 1 commit contributed to the release.
+ - 7 days passed between releases.
+ - 1 commit was understood as [conventional](https://www.conventionalcommits.org).
  - 0 issues like '(#ID)' were seen in commit messages
 
 ### Commit Details
@@ -30,6 +27,41 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 <details><summary>view details</summary>
 
  * **Uncategorized**
+    - 🏷️ Release schema-registry-api v2.0.0, safety bump schema-registry-cli v2.0.0 ([`e1bf358`](https://github.com/ilaborie/schema-registry-cli/commit/e1bf358ecc8f34464c657655322434793cb698b8))
+</details>
+
+## v1.0.0 (2023-04-23)
+
+<csr-id-25ebd97c1c43e01754b11c44ab08a0bf37286a8c/>
+
+### Chore
+
+ - <csr-id-25ebd97c1c43e01754b11c44ab08a0bf37286a8c/> 🍽️ prepare cli release
+
+### Chore
+
+ - <csr-id-1e5359254eec50b9c9712b07230a1c82ae2b1416/> 🏷️ Release schema-registry-cli v1.0.0
+
+### New Features
+
+ - <csr-id-61b026741b67ff31692965d218c89d11eb2658d3/> ✨ CLI for schema registry
+
+### Commit Statistics
+
+<csr-read-only-do-not-edit/>
+
+ - 3 commits contributed to the release.
+ - 3 commits were understood as [conventional](https://www.conventionalcommits.org).
+ - 0 issues like '(#ID)' were seen in commit messages
+
+### Commit Details
+
+<csr-read-only-do-not-edit/>
+
+<details><summary>view details</summary>
+
+ * **Uncategorized**
+    - 🏷️ Release schema-registry-cli v1.0.0 ([`1e53592`](https://github.com/ilaborie/schema-registry-cli/commit/1e5359254eec50b9c9712b07230a1c82ae2b1416))
     - 🍽️ prepare cli release ([`25ebd97`](https://github.com/ilaborie/schema-registry-cli/commit/25ebd97c1c43e01754b11c44ab08a0bf37286a8c))
     - ✨ CLI for schema registry ([`61b0267`](https://github.com/ilaborie/schema-registry-cli/commit/61b026741b67ff31692965d218c89d11eb2658d3))
 </details>

--- a/schema-registry-cli/Cargo.toml
+++ b/schema-registry-cli/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "schema-registry-cli"
-version = "1.0.0"
+version = "2.0.0"
 authors = ["Igor Laborie <ilaborie@gmail.com>"]
 description = "CLI for a schema-registry"
 license = "MIT OR Apache-2.0"
@@ -19,7 +19,7 @@ repository = "https://github.com/ilaborie/schema-registry-cli.git"
 [dependencies]
 clap = { version = "4.2", features = ["derive"] }
 clap_complete = "4.2"
-schema-registry-api = { version = "1.0.0", path = "../schema-registry-api" }
+schema-registry-api = { version = "^2.0.0", path = "../schema-registry-api" }
 serde = "1.0"
 serde_json = "1.0"
 thiserror = "1.0"

